src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java

```diff
@@ -108,6 +111,7 @@
     // -@cs[ParameterNumber] Class is immutable, we need that amount of arguments.
     public LocalizedMessage(int lineNo,
```

Comment:
Can we reuse this constructor inside [this one](https://github.com/checkstyle/checkstyle/pull/4828/files#diff-4cd17638e0ac98de46e3603a0473be44R154) ?